translations/README.md

```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) | [:ru: Russian](README.russian.md) |
```

Comment:
This doesn't show up correctly:

![image](https://user-images.githubusercontent.com/5722022/67632413-34297280-f8a3-11e9-807a-3da8e084f452.png)